FileManager/src/org/openintents/filemanager/FileManagerActivity.java

```diff
@@ -186,9 +186,8 @@ public boolean onOptionsItemSelected(MenuItem item) {
 	// The following methods should properly handle back button presses on every API Level.
 	@Override
 	public boolean onKeyUp(int keyCode, KeyEvent event) {
-		if (VERSION.SDK_INT > VERSION_CODES.DONUT) {
-			if (keyCode == KeyEvent.KEYCODE_BACK && mFragment.pressBack())
-				return true;
+		if (VERSION.SDK_INT > VERSION_CODES.DONUT && keyCode == KeyEvent.KEYCODE_BACK && mFragment.pressBack()) {
```

Comment:
I prefer the previous style to clearly state that this branch is about Android versions.